lib/js/components/ProgressBar/ProgressBarLegend.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+	<div
+		class="ds-progressBarLegend"
+		:class="{ '-ds-compact': layout === PROGRESS_BAR_LEGEND_LAYOUTS.COMPACT }"
+	>
+		<progress-bar-legend-item
+			v-for="(data, index) in ranges"
+			:key="index"
+			:size
```

Comment:
We don't use this pattern anywhere and may be a bit confusing. I would keep the long form